### PR TITLE
require phpunit for dev environment, add php 7.2 to travis

### PR DESCRIPTION
**Pull request by Gemorroj**

### CODE REVIEW

**Comment by Gemorroj** [comment]
sorry, I see that the project is not active

File: src/Linfo/OS/Linux.php [hunk 1]
```diff
@@ -1297,6 +1302,16 @@ public function getServices()
...
+        // systemd services
+        foreach ($this->settings['services']['systemdServices'] as $service => $systemdService) {
+            $command = $this->exec->exec('systemctl show -p MainPID', $systemdService);
+            $pid = str_replace('MainPID=', '', $command);
+            if ($pid != '' && is_numeric($pid)) {
+                $pids[$service] = $pid;
+            }
+        }
+
+
```
Comment (Gummibeer) [inline src/Linfo/OS/Linux.php]: one empty-line too much

**Comment by jrgp** [comment]
Sorry for the delayed response. I got an email for this a few days ago but forgot about it until I got an email notif today.

**Comment by jrgp** [comment]
hmmm 152 files changed?

**Comment by Gemorroj** [comment]
@jrgp I change the project to fit my needs (delete UI and updating the code) does not need to merge those changes.

**Pull request closed without merging**